OC-114 & OC-113 | Add an agreement checkbox & Add contracts to the annex

SnejannaTumanova opened the pull request:
Add an agreement checkbox and add linking of checkboxes to buttons on the page

SnejannaTumanova committed:
feat: add 2 contracts in web application

SnejannaTumanova committed:
feat: add linking of checkboxes to buttons on page

program-miss commented on src/hooks/useCourseActions.ts:
```diff
@@ -76,13 +96,26 @@ export function useCourseActions(course: ICourse, role: RoleType) {
-  }, [connected, isMainnet, t]);
+  }, [connected, isMainnet, hasAcceptedTermsCourse, t]);
+
+  // useEffect(() => {
```
Стоит ли удалять этот кусок кода?

SnejannaTumanova committed:
fix: delete unnecessary code

program-miss committed:
feat: add visual changes

- make link to aggrement with underline;
- rename ChekBoxInput to CheckboxInput;
- rename dataLoaded to dataLoadedFromBlockchain;
- remove unnecessary props in CheckboxInput;
- text about agreement conditions are now shown if purchase contract balance <= 0;
- increase icon size of hint for button;
- improve styles for checkbox;
